plugins/available/python.plugin.bash

```diff
@@ -1,31 +1,31 @@
-cite about-plugin
+# shellcheck shell=bash
 about-plugin 'alias "shttp" to SimpleHTTPServer'
 
-if [[ "$OSTYPE" == 'linux'* ]]
-then
-  alias shttp='python2 -m SimpleHTTPServer'
+if _command_exists python2; then
```

Comment:
why do we prefer `python2`?

Reply:
No idea on earth. 

Previously, it was deciding based on OS, which makes no sense. Mac only had python v2, and on Linux it forced it.